### ISSUE TITLE
[Bug]: PHPstorm completely freezes when editing Console/Kernel.php

### ISSUE THREAD

**Issue by kasus** [laravel-idea/plugin#609]
### Bug description

PHPStorm just stops responding when I open Console/Kernel.php and try to edit something.
So I have to use System monitor to force close the php storm and reopen. Project isn't too big and I didn't have such issue before. It doesn't crash, so I don't have any log to attach.

I am using MacBook Pro 14 with M1 pro chip

### Plugin version

5.7.3.221

### Operating system

MacOS

### Steps to reproduce

1. Open project
2. Edit something in Console/Kernel.php

**Comment by adelf** [comment]
Hi. Laravel Idea uses console kernel only during new Console command generation. To add console command to kernel (it's not needed for fresh Laravel versions). 
Does it happen if you disable Laravel Idea?

**Comment by kasus** [comment]
Yes, it still happens when I disable Laravel Idea, but when I disabled "Laravel" plugin by Daniel Espendiller, the issue is gone.
I guess I don't need both plugins?

**Comment by adelf** [comment]
Yes. Laravel Idea covers all it features.
